abi-bindings/go/Teleporter/TeleporterMessenger/event.go

```diff
@@ -86,19 +89,19 @@ func FilterTeleporterEvents(topics []common.Hash, data []byte, event string) (in
 	var out interface{}
 	switch e {
 	case SendCrossChainMessage:
-		out = new(TeleporterMessengerSendCrossChainMessage)
+		out = new(ReadableTeleporterMessengerSendCrossChainMessage)
 	case ReceiveCrossChainMessage:
-		out = new(TeleporterMessengerReceiveCrossChainMessage)
+		out = new(ReadableTeleporterMessengerReceiveCrossChainMessage)
 	case AddFeeAmount:
-		out = new(TeleporterMessengerAddFeeAmount)
+		out = new(ReadableTeleporterMessengerAddFeeAmount)
 	case MessageExecutionFailed:
-		out = new(TeleporterMessengerMessageExecutionFailed)
+		out = new(ReadableTeleporterMessengerMessageExecutionFailed)
 	case MessageExecuted:
-		out = new(TeleporterMessengerMessageExecuted)
+		out = new(ReadableTeleporterMessengerMessageExecuted)
 	case RelayerRewardsRedeemed:
 		out = new(TeleporterMessengerRelayerRewardsRedeemed)
```

Comment:
Any reason why RelayerRewardsRedeemed is different than the other cases?

Reply:
It didn't have any fields that weren't printed properly.